gpt_engineer/applications/cli/cli_agent.py

```diff
@@ -205,19 +208,18 @@ def improve(
         files_dict = self.improve_fn(
             self.ai, prompt, files_dict, self.memory, self.preprompts_holder
         )
-
-        # No need to run entrypoint for improve right?
-        # if not execution_command and ENTRYPOINT_FILE not in files_dict:
-        #     entrypoint = gen_entrypoint(
-        #         self.ai, files_dict, self.memory, self.preprompts_holder
-        #     )
-        #     combined_dict = {**files_dict, **entrypoint}
-        #     files_dict = FilesDict(combined_dict)
-
+        # entrypoint = gen_entrypoint(
```

Comment:
are we sure we want commented code here? its always in the git history if we want to bring it back later - or are we expecting this to be commented in and out regularly